enterprise/tpa_pipeline.py

```diff
@@ -142,7 +144,9 @@ def handle_redirect_after_social_auth_login(backend, user):
 
     """
     enterprise_customers_count = EnterpriseCustomerUser.objects.filter(user_id=user.id).count()
-    if enterprise_customers_count > 1:
+    next_url = backend.strategy.session_get('next')
+    using_enrolment_url = re.match(r'\/enterprise\/.*\/course\/.*\/enroll', next_url)
```

Comment:
I don't think we need to escape forward slashes `/`

Reply:
Removed